source/Ship.cpp

```diff
@@ -1306,6 +1306,7 @@ void Ship::Place(Point position, Point velocity, Angle angle, bool isDeparting)
 
 
 
+
```

Comment:
Hmmm... There was other things here, that have since been removed. When I removed them it looked like this extra line was the third. It seems it is now the 4th, so I will remove it.

Thanks!